content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -4599,6 +4599,7 @@ This push can be emitted with `user.present` set to `false` when a user writes t
 | Object      | Notes                                |
 | ----------- | ------------------------------------ |
 | `user_type` | Possible values: `agent`, `customer` |
+| `thread_id` | Empty if a user added to an inactive chat. |
```

Comment:
In such a case, thread_id will be an empty string?

Reply:
yes

Reply:
ok 👍🏻